packages/vue-components/src/Popover.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <span
+    :class="trigger === 'click' ? 'trigger-click' : 'trigger'"
+    data-mb-component-type="popover"
+    tabindex="0"
+  >
+    <portal v-if="targetEl.id" :to="targetEl.id">
+      <template #title>
```

Comment:
these are local to the scope of the template being compiled (i.e. `Popover.vue`) (try testing a trigger with a popover with both header and content)

Reply:
Thank you for the catch, I've fixed this as well as the hydration errors and checked the console (locally and on the deploy preview) for hydration warnings. Do let me know if I've missed out any other concerns!